web/app/view/map/GeofenceMap.js

```diff
@@ -68,11 +68,9 @@ Ext.define('Traccar.view.map.GeofenceMap', {
         if (this.area !== '') {
```

Comment:
Unrelated to the change, but maybe we should change this to just `if (this.area) {`?